docs/ref/models/fields.txt

```diff
@@ -1157,6 +1157,36 @@ you can use the name of the model, rather than the model object itself::
         # ...
         pass
 
+Relationships defined this way on
+:ref:`abstract models <abstract-base-classes>` are resolved when the
```

Comment:
you can replace the space in "models <abstract" with a line break for better flow.